wiki/Community/Mappers_Guild/es.md

```diff
@@ -0,0 +1,74 @@
+---
+tags:
+  - Mapper's Guild
+  - MG
+---
+
+# Mappers' Guild
+
+![](img/mg-logo.png)
+
+El **Mappers' Guild** es un grupo de mappers responsables de crear [beatmaps](/wiki/Beatmap) para los [Artistas Destacados](/wiki/Featured_Artists). Utilizando el formato de gremios estilo MMO, permite que los mappers obtengan recompensas por participar en el proceso de clasificación de los beatmaps de Artistas Destacados y cooperando en misiones de mappeo.
+
+La organización y discusión del gremio puede ser encontrada en el canal de discord `#mappers-guild` en el server de [osu!dev Discord server](https://discord.gg/ppy).
+
+## Sistema
+
+*Para la información mas actualizada sobre el Mappers' Guild, referirse a las [preguntas frecuentes de este sitio](https://mappersguild.com/faq).*
+
+Mappers con 3 o más mapas clasificados pueden unirse al gremio en cualquier momento a través de [Mappers' Guild website](https://mappersguild.com/) autenticando su cuenta de osu!. Cualquiera es libre de ver las misiones y actividades de mappeo en la página web.
+
+Los mappers pueden conseguir recompensas participando en el proceso de clasificación de mapas que sean de Artistas Destacados y estén conectados al Mappers' Guild. Esto se enfoca principalmente en la creación de beatmaps, sin embargo, las recompensas también son proveídas a usuarios que contribuyan al [moddeo](/wiki/Modding), a la creación de [storyboards](/wiki/Storyboard), y completando misiones dentro del gremio.
+
+### Bandas
+
+Las bandas son grupos de mappers que se registran para completar misiones juntos. Los mappers se pueden unir a cuantas bandas quieran en cualquier momento dado.
+
+Misiones especiales requieren un rango mayor de gremio para participar. Para una banda calificar como una de alto rango, al menos la mitad de sus miembros deben de ser del rango requerido o más alto.

```

Comment:
```suggestion
Las bandas son grupos de mappers que se registran para completar misiones juntos. Los mappers pueden unirse a tantos bandas como quieran a la vez.

Las misiones especiales requieren rangos de gremio más altos para participar. Para que una banda califique como un rango más alto, al menos la mitad de sus miembros deben tener el rango requerido o superior.
```

Reply:
did apply this fully but changed "tantos" to "tantas"

Reply:
correct